Sources/MapboxCoreNavigation/EHorizon/RoadObjectMatcher.swift

```diff
@@ -92,10 +92,16 @@ final public class RoadObjectMatcher {
 
      - parameter point: Point representing the object.
      - parameter identifier: Unique identifier of the object.
+     - parameter direction: Direction of the provided point, which is going to be matched.
      */
-    public func match(point: CLLocationCoordinate2D, identifier: RoadObjectIdentifier) {
-        let point = MatchableGeometry(id: identifier, coordinates: [point].map(CLLocation.init))
-        native.matchPoints(forPoints: [point], useOnlyPreloadedTiles: false)
+    public func match(point: CLLocationCoordinate2D, identifier: RoadObjectIdentifier, direction: CLLocationDirection? = nil) {
```

Comment:
Consider replacing CLLocationDirection with CLHeading for forward compatibility in case we ever need to consider the heading accuracy as part of matching. For that matter, even if we leave it as a CLLocationDirection, `heading` would be a less ambiguous label than `direction`, which can also refer to a map rotation direction or a relative direction. (A CLLocation would probably be a more correct representation of a coordinate plus a heading, even if some of the fields happen to be ignored for the time being.)

Reply:
Thanks, even though `CLLocation` usage is tempting it'll require public API change. Since third parameter uses default value it seems more reasonable to me. So I've replaced it to `CLHeading` for now.